libvirt.go

```diff
@@ -294,6 +299,24 @@ func (d *Driver) Create() error {
 	return d.Start()
 }
 
+func createImage(src, dst string) error {
+	// #nosec
+	cmd := exec.Command("qemu-img",
+		"create",
+		"-f", "qcow2",
+		"-o", fmt.Sprintf("backing_file=%s", src),
+		dst)
+	if _, err := exec.LookPath("qemu-img"); err != nil {
+		cmd = exec.Command("cp", "-f", src, dst)
```

Comment:
I'd favour
```
        if err := mcnutils.CopyFile(src, dst); err != nil {                                                                                                                                                                                                  
            return err
        }
```

Reply:
I will benchmark it but I think `cp` wins by large

Reply:
``` 
/usr/bin/cp took: 29.140343565s
mcnutils.CopyFile took: 47.625550974s
```

Reply:
We can decide to use `cp`, but this should be done in `CopyFile` so that it benefits all users.

Reply:
Unexpected to  see such a big difference as `mcnutils.CopyFile` is not doing a lot more than calling https://golang.org/pkg/io/#Copy

Reply:
Will it work for Windows ? I'm not sure.

Reply:
On 2 different machines, I'm seeing `mcnutils.CopyFile` being comparable to `cp` with respect to copying time, so it seems premature to replace it.
```
CopyFile: 24.008310
cp: 22.868998
```

```
CopyFile: 1.600820
cp: 1.500317
```
(first test is with qcow2 image, second is with a compressed bundle, before each copy I ran `sudo sh -c 'echo 3 >/proc/sys/vm/drop_caches'`, so I only tested the 'cold cache' case)

Reply:
Looping 10 times without dropping the cache, I get better average with `CopyFile`

Reply:
I tried 2 more times and CopyFile took 57 and 59s. Then tried with cp, 27s, then 1min5. I really don't understand.

Anyway, most of us will have qemu-img :)